Dockerfile

```diff
@@ -61,8 +61,8 @@ ENV BUILD_TARGET "DEBUG"
 
 ENV PYTHON_COMMAND python3
 
-RUN git clone "https://github.com/tianocore/edk2-libc" libc
-RUN git clone "https://github.com/tianocore/edk2-platforms" platforms
+RUN git clone --depth=1 "https://github.com/tianocore/edk2-libc" libc
+RUN git clone --depth=1 "https://github.com/tianocore/edk2-platforms" platforms
```

Comment:
Here we should remove `--depth=1`, otherwise we leave no flexibility to which version of `libc` and `platforms` to use.